index.js

```diff
@@ -136,6 +137,7 @@ var avro_decoded_data = function (schema_data, record) {
   const type = avro.parse(schema_data)
   var decoded = new Buffer(record, 'base64')
   var verify = type.fromBuffer(decoded)
+  console.log(decoded)
```

Comment:
@EdwinGuzman,

I removed it. That was there mainly for troubleshooting purposes. When I do proper logging, I am going to have logging only where ever it is required.